functions.php

```diff
@@ -68,6 +68,47 @@ function course_theme_init() {
 			'label' => __( 'Button', 'course' ),
 		)
 	);
+	register_block_style(
+		'core/group',
+		array(
+			'name'  => 'group-left-border',
+			'label' => __( 'Left Border', 'course' ),
+		)
+	);
+	register_block_style(
+		'core/image',
+		array(
+			'name'  => 'course-image-border',
+			'label' => __( 'Image Border', 'course' ),
+		)
+	);
+
+	$block_pattern_categories = array(
+		'sensei-lms' => array( 'label' => __( 'Sensei LMS', 'course' ) )
```

Comment:
Why do we need to translate `Sensei LMS` ?

Reply:
Removed it here https://github.com/Automattic/sensei-theme/pull/44/commits/73331765814cfc083618f8a463f16acd60d6cde1